src/test/play-2/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/addtoalist/addToSummary.scala.html

```diff
@@ -0,0 +1,40 @@
+@import uk.gov.hmrc.hmrcfrontend.views.html.components._
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.addtoalist.ItemType
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.addtoalist.ListItem
+@import uk.gov.hmrc.govukfrontend.examples.MessagesSupport
+@import play.api.test.FakeRequest
+
+@this(hmrcAddToAList: HmrcAddToAList)
+
+@()
+
+@hmrcAddToAList(AddToAList(
+    itemList = Seq(
+        ListItem(
+            "Sydney Russell",
+            "#",
+            "#"
+        ),
+        ListItem(
+            "Michael Cain",
+            "#",
+            "#"
+        ),
+        ListItem(
+            "Jeremy Winter",
+            "#",
+            "#"
+        ),
+        ListItem(
+            "Elizabeth Kean",
+            "#",
+            "#"
+        )
+    ),
+    itemType = Some(ItemType(
+        "director",
+        "directors"
+    )),
+    hintText = Some("You must tell us about all your directors."),
+    formAction = Some("")
+))(MessagesSupport().messages, FakeRequest())
```

Comment:
@JoPintoPaul I wonder if seeing this in an example some developers might get confused. Would it be possible for us to add an implicit messages / request to the class that's executing any tests of these? so the examples are closer to expected usage

Reply:
@oscarduignan Updated the PR with your suggestion.